extensions/wikia/PortableInfoboxBuilder/services/PortableInfoboxBuilderHelper.class.php

```diff
@@ -9,20 +9,15 @@ class PortableInfoboxBuilderHelper {
 	/**
 	 * Checks if template is classified as an infobox
 	 *
-	 * @param $title
+	 * @param $title Title
 	 * @return bool
 	 */
 	public static function isInfoboxTemplate( $title ) {
-		$tc = new TemplateClassificationService();
-		$isInfobox = false;
-
-		try {
-			$type = $tc->getType( F::app()->wg->CityId, $title->getArticleID() );
-			$isInfobox = ( $type === TemplateClassificationService::TEMPLATE_INFOBOX );
-		} catch ( Swagger\Client\ApiException $e ) {
-			// If we cannot reach the service assume the default (false) to avoid overwriting data
+		if ( $title->getNamespace() === NS_TEMPLATE ) {
```

Comment:
You could use ``inNamespace()`` method available in title object, and make it a single return statement.